examples/nuclio/ingest/ingest_example_test.go

```diff
@@ -13,9 +17,14 @@ func TestIngestIntegration(t *testing.T) {
 	if testing.Short() {
 		t.Skip("Skipping integration test.")
 	}
+	v3ioConfig, err := config.LoadConfig(filepath.Join("../../../", config.DefaultConfigurationFileName))
```

Comment:
`filepath.Join` takes as many arguments as you wish, use it!

Reply:
store test resources and configuration at `testdata` directory